docs/changelog/1790.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add ``exec`` subcommand that allows users to run arbitrary command within the tox environment (without needing to modify
```

Comment:
```suggestion
Add ``exec`` subcommand that allows users to run an arbitrary command within the tox environment (without needing to modify
```

Reply:
Is it supposed to run exactly one command?

I tried some variations of quoting to run two commands, but failed :-)

Reply:
Yeah, single run only. Hence in the description ``execute an arbitrary command``.